### PR TITLE
update translations

### PR DESCRIPTION
**Pull request by r10s**

### CODE REVIEW

File: de/serverguide.md [hunk 1]
```diff
@@ -1,67 +1,47 @@
...
-- your server has the IPv4 address 24.48.100.24 - you can find out with the
-  command `ip a` and look for a similar-looking number (which doesn't start
-  with 127 or 172).
-- your server has the IPv6 address 7fe5:2f4:1ba:2381::3 (you can find it in `ip
-  a`, 2 lines below the IPv4 address. Ignore the `/64` at the end. Don't use
-  the one starting with `fe80`, it doesn't count).
+Nehmen wir an:
+- Sie haben example.org gekauft. Im Moment wollen Sie nur einen Mailserver, aber Sie überlegen, später eine Website unter https://example.org zu hosten.
+- Ihr Server hat die IPv4-Adresse 24.48.100.24
+- Sie können diePv4-Adresse  mit dem Befehl `ip a` herausfinden und nach einer ähnlich aussehenden Nummer suchen (die nicht mit mit 127 oder 172 beginnt).
```
Comment (missytake) [inline de/serverguide.md]:
```suggestion
- Ihr Server hat die IPv4-Adresse 24.48.100.24 - Sie können die echte IPv4-Adresse  mit dem Befehl `ip a` herausfinden und nach einer ähnlich aussehenden Nummer suchen (die nicht mit mit 127 oder 172 beginnt).
```

That wasn't supposed to be a bullet point, just a hyphen^^

File: de/serverguide.md [hunk 2]
```diff
@@ -74,29 +54,23 @@ Now you could configure the domain settings for example.org like this:
 | TXT   | @               | "v=spf1 mx -all"                                     | 5min |          |
 | TXT   | _dmarc          | v=DMARC1;p=quarantine;rua=mailto:mailadm@example.org | 5min |          |
 
-You can setup the DKIM key after setting up mailcow,
-in System>Configuration>Options>ARC/DKIM keys.
+Sie können den DKIM-Schlüssel nach der Einrichtung von mailcow unter System / Konfiguration / Optionen / ARC/DKIM-Schlüssel einrichten.
 
-You can do more than 5 minutes, but in case you notice something is wrong a
-short time helps with fixing the wrong entry.
+Sie können mehr als 5 Minuten brauchen, aber wenn Sie merken, dass etwas falsch ist, hilft eine kurze Zeitspanne bei der Korrektur des falschen Eintrags.
```
Comment (missytake) [inline de/serverguide.md]:
```suggestion
Sie können mehr als 5 Minuten einstellen, aber wenn Sie merken, dass etwas falsch ist, hilft eine kurze Zeitspanne bei der Korrektur des falschen Eintrags.
```

File: de/serverguide.md [hunk 3]
```diff
@@ -267,61 +217,45 @@ echo "alias mailadm=$PWD/scripts/mailadm.sh" >> ~/.bashrc
...
-## Optional: Redirect all HTTP traffic to HTTPS
+## Optional: Den gesamten HTTP-Verkehr auf HTTPS umleiten
 
-By default,
-the nginx server also responds unencrypted
-on port 80.
-This can be bad,
-as some users might enter passwords
-over this unencrypted connection.
+Standardmäßig, antwortet der nginx-Server auch unverschlüsselt auf Port 80. Das kann schlecht sein, da einige Benutzer möglicherweise Passwörter über diese unverschlüsselte Verbindung eingeben.
```
Comment (missytake) [inline de/serverguide.md]:
```suggestion
Standardmäßig antwortet der nginx-Server auch unverschlüsselt auf Port 80. Das kann schlecht sein, da einige Benutzer möglicherweise Passwörter über diese unverschlüsselte Verbindung eingeben.
```

Not so important.

File: de/serverguide.md [hunk 4]
```diff
@@ -340,24 +274,17 @@ server {
 }
 ```
 
-Then apply the changes with `sudo docker compose restart nginx-mailcow`.
+Dann wenden Sie die Änderungen mit `sudo docker compose restart nginx-mailcow` an.
 
-## Optional: No Logs, No Masters
+## Optional: Keine Protokolle, keine Master
```
Comment (missytake) [inline de/serverguide.md]:
```suggestion
## Optional: Keine Protokolle, keine Herren
```
To be fair - the saying is not really translatable. Not a blocker.

**Review by missytake — APPROVED** [review]
I can't judge the polish translation, but it should be fine I guess, at least the code doesn't seem to be changed (except that they translated "example" - but whoever reads it needs to exchange that anyway, and as long as there is no *full* translation that part needs to be understood nevertheless. not optimal, but not sure if worth fixing on transifex).

Apart from that, the translation is okay overall :) Some things could be changed, but only if it's not a hassle with transifex.

File: de/serverguide.md (hunk 4 above)
Comment (r10s) [reply de/serverguide.md]: i leave it untranslated. however, such "jokes" are hard for translators without additional hints. this headline may look weird in the other translations as well (you can add translations hints to transifex (you can log in with your github handle there))

**Comment by r10s** [comment]
thanks for review!

**Pull request merged**